data/items/items.xml

```diff
@@ -706,15 +706,17 @@
 	<item id="1497" article="a" name="magic wall">
 		<attribute key="type" value="magicfield" />
 		<attribute key="decayTo" value="0" />
-		<attribute key="duration" value="20" />
+		<attribute key="durationMin" value="14" />
+		<attribute key="durationMax" value="20" />
```

Comment:
What about `<attribute key="duration" minvalue="14" maxvalue="20" />`, or even `min` and `max`?

Reply:
It doesn't seem like a bad idea, I like the way you think

Reply:
but how would it work with the remaining items that use duration? 3 parameters?

Reply:
It's either `value`, or `min` and `max`. If the item specifies `value`, don't even try to read the others.

Reply:
Maybe value, minValue and maxValue. Somehow i don't like how its called value then it just min or max.

Reply:
I suggest duration and durationMax - would be consistent with count and countMax in monsters loot